react/components/admin/pages/Form/SelectConditions.tsx

```diff
@@ -160,7 +160,7 @@ class SelectConditions extends React.Component<Props> {
     return (
       <DatePicker
         value={values && values.date}
-        onChange={date => {
+        onChange={(date: string) => {
```

Comment:
Is this really a string? 🤔

Reply:
I have absolutely no idea why I did that. I'll change it right away.